include/lsst/meas/base/NaiveCentroid.h

```diff
@@ -46,7 +46,8 @@ class NaiveCentroidControl {
     LSST_CONTROL_FIELD(background, double, "Value to subtract from the image pixel values");
     LSST_CONTROL_FIELD(doFootprintCheck, bool, "Do check that the centroid is contained in footprint.");
     LSST_CONTROL_FIELD(maxDistToPeak, double,
-                       "If set > 0, Centroid Check also checks distance from footprint peak.");
+                       "If >0; maximum distance in pixels between the footprint peak and centroid allowed before "/
```

Comment:
Have you run `scons` on this? This won't compile, as it's not valid C++. No `/` there; the compiler will just combine the strings without any other characters needed.

Reply:
flake8 only lints python, not C++

Reply:
Fixed the formatting. Scons compiled it without complaint.